src/components/v1/V1Project/ProjectActivity/index.tsx

```diff
@@ -65,6 +65,14 @@ export default function ProjectActivity() {
         key: 'useAllowanceEvent',
         value: null, // Exclude all useAllowanceEvents, no UI support yet
       },
+      {
+        key: 'distributeToTicketModEvent',
```

Comment:
adding some events we should exclude since we don't show these directly in project activity list